sections/conclusion/conclusion-chapter.tex

```diff
@@ -20,6 +20,9 @@ \subsection{Future work}
 \\
 Additionally, we have experienced that not all contexts are equal in terms of importance.
 Due to this, it could be beneficial to include an attention mechanism that is able to learn the importance of each context or context combination for the convolution layer.
+This kind of attention idea could also be applied to layers.
```

Comment:
burde hele den her sektion, altså linje 21 til 25 være i konklusionen? hvorfor spekulerer vi om noget i konklusionen i stedet for bare at skrive det i future work elelr diskussion

Reply:
loool

Reply:
xd